pvlib/modelchain.py

```diff
@@ -1740,6 +1751,19 @@ def run_model_from_effective_irradiance(self, data=None):
 
         Notes
         -----
+        Optional `data` columns ``'cell_temperature'``,
+        ``'module_temperature'`` and ``'poa_global'`` are used for determining
+        cell temperature.
```

Comment:
not rendering as desired. might need a blank line here

https://pvlib-python--1129.org.readthedocs.build/en/1129/generated/pvlib.modelchain.ModelChain.run_model_from_effective_irradiance.html